arrow/arrow.py

```diff
@@ -517,6 +518,8 @@ def span(
         :param exact: (optional) whether to have the start of the timespan begin exactly
             at the time specified by ``start`` and the end of the timespan truncated
             so as not to extend beyond ``end``.
+        :param weekday: (optional) only in combination with frame == "week". Conform with self.isoweekday(), 1 = Monday,
```

Comment:
I tend to think we should call this `week_start` for clarity.

Reply:
Hey,
I changed the parameter name to week_start.